### DISCUSSION TITLE
DRAMClocks

### DISCUSSION THREAD

**Discussion by AnaKhorguani29** [opcm/pcm#355] [Q&A] [answered]
Hello, I am running `./pcm-power.x` for power usage evaluation. I got a bit confused about `DRAMClocks`. Does it represent DRAM memory frequency in Hz? 

Given a Configured Memory Speed - 2666 MT/s, should the expected result be around 1333,000,000? (Half because of Double Data Rate thing)

Thank you in advance.

**Comment by opcm** [comment, marked as answer]
yes, correct, the number should be the half

**Reply by AnaKhorguani29** [reply]
Thank you very much for the confirmation.

I tested on two different machines, and I got the expected result on one of them. But on the other machine I got way lower `DRAMClocks`, which additionally fluctuated a lot. And since I read that DRAM frequency does not change dynamically, I was unsure that I understood this counter correctly. 

Thanks again for such a quick reply.
